modules/shared_image_gallery/packer_build/packer.tf

```
@@ -151,5 +151,5 @@ locals {
   managed_local_identity  = try(var.managed_identities[var.client_config.landingzone_key][var.settings.managed_identity_key].id, "")
   managed_remote_identity = try(var.managed_identities[var.settings.lz_key][var.settings.managed_identity_key].id, "")
   provided_identity       = try(var.settings.managed_identity_id, "")
-  managed_identity        = try(merge(local.managed_local_identity, local.managed_remote_identity, local.provided_identity), [])
+  managed_identity        = try(coalesce(local.managed_local_identity, local.managed_remote_identity, local.provided_identity), [])
```

```suggestion
  managed_identity        = concat(local.managed_local_identity, local.managed_remote_identity, local.provided_identity)
```

Can you please check with that suggestion?

---

I've tried that but get the error: _"Invalid value for "seqs" parameter: all arguments must be lists or tuples; got string."_ I've chosen to use coalesce because the use case for multiple managed identities is very minor and merge was giving errors during the conversion to json and use by packer.